guis/common/db_classes/straw_location.py

```diff
@@ -69,18 +70,20 @@ class StrawLocation(BASE, OBJECT):
     ## INITIALIZATION ##
 
     def __init__(
-        self, location_type=None, number=int(), pallet_id=None, create_key=None
+        self, location_type=None, number=int(), pallet_id=None, create_key=None, check_key=True, timestamp=time()
```

Comment:
instead of bypassing the check key, instead try to make the procedure and straw_location all at once:

```
        p = Procedure.StrawProcedure(
            process=process, pallet_id=pallet_id, pallet_number=pallet_number
        ) 
```